docs/index.md

```diff
@@ -20,6 +20,44 @@ for working with [Google Cloud Storage](https://cloud.google.com/storage/)
 Common operations are exposed via the
 [`StorageClient`](obj/api/Google.Storage.V1.StorageClient.yml) class.
 
+Quick start:
+```bat
+PM> Install-Package Google.Storage.V1 -Pre
+```
+
+Sample code:
+```csharp
```

Comment:
That works for me. Happy to move my snippets that way. Do you have an infrastructure that I can plug into?

On Apr 18, 2016, at 11:40 AM, Jon Skeet <notifications@github.com<mailto:notifications@github.com>> wrote:


In docs/index.md<https://github.com/GoogleCloudPlatform/gcloud-dotnet/pull/118#discussion_r60111746>:

> @@ -20,6 +20,44 @@ for working with [Google Cloud Storage](https://cloud.google.com/storage/)
>  Common operations are exposed via the
>  [`StorageClient`](obj/api/Google.Storage.V1.StorageClient.yml) class.
>
> +Quick start:
> +```bat
> +PM> Install-Package Google.Storage.V1 -Pre
> +```
> +
> +Sample code:
> +```csharp


We should be able to include this code using a Markdown reference to the snippet. Will get to that when we've done everything else.

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub<https://github.com/GoogleCloudPlatform/gcloud-dotnet/pull/118/files/f2279a6807fb057107949cbabca036d0ced8a75a#r60111746>